src/javascript/app/base/header.js

```diff
@@ -547,21 +561,74 @@ const Header = (() => {
+            high_risk_accounts_accordion_header.style.display = 'none';
+            low_risk_non_eu_accordion_header.style.display    = 'flex';
+            low_risk_eu_accordion_header.style.display        = 'flex';
+            $('<div class="account__switcher-seperator" />').insertBefore('#account__switcher-accordion-eu');
+        }
+    };
+
+    const mapCurrencyName = (currency) => {
+        let currencyName;
+        if (currency === 'USD') {
+            currencyName = 'US Dollar';
+        } else if (currency === 'EUR') {
+            currencyName = 'Euro';
+        } else if (currency === 'GBP') {
+            currencyName = 'Pound Sterling';
+        } else if (currency === 'AUD'){
+            currencyName = 'Australian Dollar';
+        } else if (currency === 'BTC'){
+            currencyName = 'Bitcoin';
+        } else if (currency === 'ETH') {
+            currencyName = 'Ethereum';
+        } else if (currency === 'USDT') {
+            currencyName = 'Tether Omni';
+        } else if (currency === 'eUSDT') {
+            currencyName = 'Tether ERC20';
+        } else if (currency === 'USDC') {
+            currencyName = 'USD Coin';
+        } else if (currency === 'LTC'){
+            currencyName = 'Litecoin';
+        } else {
+            currencyName = currency;
+        }
+        return currencyName;
+    };
+
     const populateAccountsList = () => {
         if (!Client.isLoggedIn()) return;
-        BinarySocket.wait('authorize', 'website_status', 'balance').then(() => {
-            const loginid_real_select = createElement('div');
-            const loginid_demo_select = createElement('div');
+        BinarySocket.wait('authorize', 'website_status', 'balance', 'landing_company', 'get_account_status').then(() => {
+            bindHeaders();
+            const loginid_non_eu_real_select   = createElement('div');
+            const loginid_eu_real_select       = createElement('div');
+            const loginid_demo_select          = createElement('div');
             Client.getAllLoginids().forEach((loginid) => {
                 if (!Client.get('is_disabled', loginid) && Client.get('token', loginid)) {
-                    const is_real        = /undefined|gaming|financial/.test(Client.getAccountType(loginid)); // this function only returns virtual/gaming/financial types
-                    const currency       = Client.get('currency', loginid);
-                    const getIcon        = (() => {
+                    const is_eu                = loginid.startsWith('MF');
+                    const is_non_eu            = loginid.startsWith('CR');
+                    const is_real              = /undefined|gaming|financial/.test(Client.getAccountType(loginid)); // this function only returns virtual/gaming/financial types
+                    const currency             = Client.get('currency', loginid);
+                    let currencyName           = mapCurrencyName(currency);
```

Comment:
```suggestion
                    let currencyName           = displayed_currencies[currency];
```
will work after adding a displayed_currencies as per suggestion here: https://github.com/deriv-com/binary-static/pull/530/files#r1162548511🙂